### ISSUE TITLE
Android Studio Flutter Inspector toggle select widget mode on is not redirecting me to where the actual source code is located to edit the code!!

### ISSUE THREAD

**Issue by AradhyaNepal** [flutter/devtools#5689]
![image](https://user-images.githubusercontent.com/66102711/233344156-c5ff2b48-d4d5-4ae4-89df-965628b17c29.png)

When a developer selects a widget from Flutter Inspector with toggle select widget mode on, in device it shows which portion of UI is selected. But developer also needs to know where the code is actually located, so that they can start editing the code as per their need. Which improves the debugging experience of developer rather than searching the code throwout the messy codebase, without it, its really hard for me as a developer to select a widget and not able to know where the code is located.

Note: I am using Android Studio, this feature to tap a widget in Flutter inspector and open the source code to start editing is available in visual studio.

Also if the feature is already available, appolozies, but I didn't found well documentation from internet about how to do in android studio.

Thank You.
___
**DevTools version**: 2.20.1
**Connected Device**:
CPU / OS: arm64 (64 bit) android
Dart Version: 2.19.1
Flutter Version: 3.7.1 / stable
Framework / Engine: 7048ed95a5 / 800594f1f4
Connected app type: Flutter native (debug build)

**Comment by kenzieschmoll** [comment]
This is the current behavior in Android Studio, and if it is not working that is a bug. @helin24 @stevemessick are either of you getting reports that this jump-to-source behavior is broken?

**Comment by stevemessick** [comment]
No, I don't recall any reports about this. I just tried it, using the embedded inspector, and it works fine. @AradhyaNepal are you clicking in the Widget Tree or the Widget Details Tree? It works in the Widget Tree. I'm not sure if it is supposed to work in the Widget Details Tree.

**Comment by AradhyaNepal** [comment]
After reading your response, I learned that the name of this feature is  jump-to-source, so I started searching it, and by using correct keyword used to search in google, I found this stackoverflow: https://stackoverflow.com/a/69708231

In my entire 1.5 years of flutter journey, I had used **space** on the folder where the project is stored:
Practice Project
-Animation

Office Project
-Project 1
-Project 2

Except me, all of my college friend and office colleagues uses visual studio code, so for 1.5 years I thought this feature is only available in visual studio code.
But the problem was caused by any **blanks in the full project path**.

After removing the blank space this feature worked.
For now the problem is solved thanks for you giving me exact keyword to search from google.

But I think this space making jump to source not work in android studio problem must be solved.

Thanks!!
